Cargo.toml

```diff
@@ -5,20 +5,15 @@ members = [
     "gateway-sp-comms",
 ]
 
-default-members = [
-    "faux-mgs",
+default-members = ["faux-mgs",
```

Comment:
Formatting typo maybe?

Reply:
I had some autoformatter that did some ... stuff...